L/libjulia/common.jl

```diff
@@ -183,6 +198,8 @@ function build_julia(ARGS, version)
         LIBUNWIND:=-losxunwind
         JCPPFLAGS+=-DLIBOSXUNWIND
     EOM
+    else
+        echo "USECLANG=1" >>Make.user
```

Comment:
Why is this needed? Can you add a comment.

Reply:
To be honest, I don't know -- I simply copied it from the script for Julia <= 1.6 above. I can try removing it to see what happens.

Reply:
Seems it works without this, so I'll remove it